src/views/data.ts

```diff
@@ -449,6 +537,98 @@ const data = [
+                    "singer": "Ayushmann Khurrana",
+                    "language": "punjabi",
+                    "genre": "romance",
+                    "link": "https://www.youtube.com/watch?v=JFYCc577kjQ"
+                }
+            
+            ]`,
+            },
+            {
+                route: "GET /songs?genre=pop",
+                content: "Returns song data from perticular genre.",
+                output: `[
+
+              {
+                  "name": "Dope Shope",
+                  "singer": "Yo Yo Honey Singh",
+                  "language": "punjabi",
+                  "genre": "pop",
+                  "link": "https://www.youtube.com/watch?v=NrXdauEv9HY"
+              },
+              {
+                  "name": "Oonchi Hai Building",
+                  "singer": "Anu Malik",
+                  "language": "hindi",
+                  "genre": "pop",
+                  "link": "https://www.youtube.com/watch?v=QmHTmr8Kazs"
+              },
+              {
+                  "name": "Brown Munde",
+                  "singer": "AP DHILLON",
+                  "language": "punjabi",
+                  "genre": "pop",
+                  "link": "https://www.youtube.com/watch?v=VNs_cCtdbPc"
+              }
+          
+          ]`,
+            },
+            {
+                route: "GET /songs?language=punjabi",
+                content: "Returns song data from perticular language.",
+                output: `[
+                  {
+                      "name": "Chan Kitta",
+                      "singer": "Ayushmann Khurrana",
+                      "language": "punjabi",
+                      "genre": "romance",
+                      "link": "https://www.youtube.com/watch?v=JFYCc577kjQ"
+                  },
+                  {
+                      "name": "Dope Shope",
+                      "singer": "Yo Yo Honey Singh",
+                      "language": "punjabi",
+                      "genre": "pop",
+                      "link": "https://www.youtube.com/watch?v=NrXdauEv9HY"
+                  },
+                  {
+                      "name": "Tunak Tunak Tun",
+                      "singer": "Daler Mehndi",
+                      "language": "punjabi",
+                      "genre": "indian pop",
+                      "link": "https://www.youtube.com/watch?v=vTIIMJ9tUc8"
+                  },
+                  {
+                      "name": "Brown Munde",
+                      "singer": "AP DHILLON",
+                      "language": "punjabi",
+                      "genre": "pop",
+                      "link": "https://www.youtube.com/watch?v=VNs_cCtdbPc"
+                  }
+              
+              ]`,
+            },
+            {
+                route: "GET /songs?artist=A.R. Rahman&genre=sufi&language=urdu",
+                content: "Return dummy songs data after chaining multiple song query parameter.",
+                output: `[
+                {
```

Comment:
Add a route where you are querying only based on two things. Say `genre` and `language`.

Reply:
Done! Plz review